SourceHgWeb/SourceHgWeb.php

```diff
@@ -268,7 +268,7 @@ private function commit_changeset( $p_repo, $p_input, $p_branch='' ) {
 
 			$t_changeset->author_email = empty($t_commit['author_email'])? '': $t_commit['author_email'];
 
-			preg_match_all('#diff[\s]*-r[\s]([^\s]*)[\s]*-r[\s]([^\s]*)[\s]([^\n]*)\n(Binary file[\s]([^\s]*)[\s]has changed|\-{3}[\s](/dev/null)?[^\t]*[^\n]*\n\+{3}[\s](/dev/null)?[^\t]*\t[^\n]*)#', $p_input, $t_matches, PREG_SET_ORDER);
+			preg_match_all('#diff[\s]*-r[\s]([^\s]*)[\s]*-r[\s]([^\s]*)[\s]([^\n]*)\n(Binary file[\s]([^\n]*)[\s]has changed|\-{3}[\s](/dev/null)?[^\t]*[^\n]*\n\+{3}[\s](/dev/null)?[^\t]*\t[^\n]*)#', $p_input, $t_matches, PREG_SET_ORDER);
```

Comment:
I'm not sure whether this would actually be an issue or not (I don't use Hg), but considering that `\s` matches HT (9), LF (10), FF (12), CR (13), and space (32), your change actually goes beyond what the PR's description says it does (i.e. it allows more than just _space_), and I'm wondering if this could be problematic in some situations, e.g. with CR+LF line endings on Windows.